src/components/Share/style.scss

```diff
@@ -0,0 +1,30 @@
+.share {
+  position: relative;
+  bottom: 2px;
+  left: 5px;
+}
+
+.facebook {
+  background-image: url(../../../static/images/FB-grn.png);
```

Comment:
Hi, any advice on how to use it within this project? It's quite complicated if I do it from webpack, as it will cause other fonts to stop working.